manifest.json

```diff
@@ -1,12 +1,12 @@
 {
-  "domain": "nextcloudtalk",
+  "domain": "https://github.com/p-nn/HANotifyNextCloudTalk",
   "name": "NextCloud Talk",
-  "documentation": "https://github.com/noxiii/HANextcloudTalk",
-  "version": "2021.12",
```

Comment:
Hey, :)
It look like that your merge request should for the fork of p-nn. There is to many changes and the version exist in my version.

https://github.com/p-nn/ha-nextcloud-talk-client